handy.h

```diff
@@ -1822,7 +1824,9 @@ END_EXTERN_C
     /* The following are not fully accurate in the above-ASCII range.  I (khw)
      * don't think it's necessary to be so for the purposes where this gets
      * compiled */
-#   define isQUOTEMETA_(c)      (FITS_IN_8_BITS(c) && ! isWORDCHAR_L1(c))
+#   if defined(PERL_CORE) || defined(PERL_IN_XS_APITEST)
+#     define isQUOTEMETA(c)      (FITS_IN_8_BITS(c) && ! isWORDCHAR_L1(c))
+#   endif
 
```

Comment:
I hadn't realised handy.h had this split between H_PERL and non-H_PERL.

Does anything test the non-H_PERL side?

Reply:
No.  This is used for bootstrapping on a type of platform that doesn't have a working perl.  I used it to get things going on an EBCDIC box when we had nothing to start from.  I don't remember the details; and I haven't needed it since